ui/app/helpers/utils/transactions.util.js

```diff
@@ -43,7 +58,19 @@ const registry = new MethodRegistry({ provider: global.ethereumProvider })
     const fourBytePrefix = prefixedData.slice(0, 10)
 
     try {
-      const sig = await registry.lookup(fourBytePrefix)
+      let fourByteSig
+      try {
+        fourByteSig = getMethodFrom4Byte(fourBytePrefix)
+      } catch (e) {
+        fourByteSig = Promise.resolve(null)
+        console.log(e)
```

Comment:
Can we use the `log` module here?

Reply:
Or the comment above addresses this as well

Reply:
@whymarrh Could we rely on the existing try-catch block. So the whole thing would become:

```
    try {
      const fourByteSig = getMethodFrom4Byte(fourBytePrefix)

      let sig = await registry.lookup(fourBytePrefix)

      if (!sig) {
        sig = await fourByteSig
      }

      if (!sig) {
        return {}
      }

      const parsedResult = registry.parse(sig)

      return {
        name: parsedResult.name,
        params: parsedResult.args,
      }
    } catch (error) {
      log.error(error)
      const tokenData = getTokenData(data)
      const { name } = tokenData || {}
      return { name }
    }
```